kong/llm/drivers/azure.lua

```diff
@@ -115,7 +115,7 @@ function _M.configure_request(conf)
   local query_table = kong.request.get_query()
 
   -- technically min supported version
-  query_table["api-version"] = conf.model.options and conf.model.options.azure_api_version or "2023-05-15"
+  query_table["api-version"] = kong.request.get_query_arg("api-version") or (conf.model.options and conf.model.options.azure_api_version) or "2023-05-15"
```

Comment:
nit: line length

Reply:
Fixed in 06c6d7f555deb5f1a9fadd4d7825b9501df595f0